### PR TITLE
[#36] Add original rank for K74R

### PR DESCRIPTION
**Pull request by meatballs**

### CODE REVIEW

**Comment by meatballs** [comment]
Yep. I'll sort that and add a couple of tests

**Comment by drvinceknight** [comment]
> Yep. I'll sort that and add a couple of tests

Perhaps you can add a doctest of the README file "in to" pytest?

**Comment by meatballs** [comment]
> Perhaps you can add a doctest of the README file "in to" pytest?

It already does. I just hadn't run it!

**Comment by drvinceknight** [comment]
Nice!

File: tests/test_strategies.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+def test_original_rank():
+    ranks = [
+        details['original_rank']
+        for details in characteristics.values()
+        if details['original_rank'] is not None]
+
+    assert len(ranks) == 63
+
+    # Test that are no duplicates in the rank values
+    assert len(ranks) == len(set(ranks))
```
Comment (drvinceknight) [inline tests/test_strategies.py]: This and the following test could just be replaced with:
```
sorted(ranks) = list(range(1, 63 + 1))
```

Comment (meatballs) [inline tests/test_strategies.py]: nice!

**Pull request merged**